### ISSUE TITLE
Library doesn't follow official naming conventions

### ISSUE THREAD

**Issue by dunca** [Kotlin/dataframe#383]
The ["Choose good names" section](https://kotlinlang.org/docs/coding-conventions.html#choose-good-names) of the Kotlin Coding Conventions documentation, mentions that acronyms longer than two characters shouldn't be capitalized.
By capitalizing CSV in class, function, etc. names, the library doesn't follow standard Kotlin naming conventions.

Actual examples from the codebase (list is not exhaustive):
_org.jetbrains.kotlinx.dataframe.io.CSV
org.jetbrains.kotlinx.dataframe.io.CSVType
org.jetbrains.kotlinx.dataframe.io.DataFrame.Companion.readCSV
org.jetbrains.kotlinx.dataframe.io.DataFrame.Companion.writeCSV_

Expected:
_org.jetbrains.kotlinx.dataframe.io.Csv
org.jetbrains.kotlinx.dataframe.io.CsvType
org.jetbrains.kotlinx.dataframe.io.DataFrame.Companion.readCsv
org.jetbrains.kotlinx.dataframe.io.DataFrame.Companion.writeCsv_

**Comment by zaleslaw** [comment]
@dunca Yes, sometimes naming violates a little bit naming conventions for acronyms, but from the other side we could have our own approach to name IO sources; personally, I prefer to use CSV instead of Csv and HTML instead of Html, I think it looks prettier and clearer

**Comment by dunca** [comment]
@zaleslaw, is there even a point in having language-official naming conventions, if they're not followed by official libraries?

**Comment by zaleslaw** [comment]
Probably, you are right, especially regarding what is pretty or not. Of course , it's subjective, as anming conventions is agreed average of the leading subjectives.

Thank you for lighting the topic

**Comment by zaleslaw** [comment]
Also, @dunca if you tasted the library, do you have something to share yet, it will be great to read, regarding interfaces/docs, how is easy to start. 

It will be great to read, you could it via issues or discussion thread if it's huge

**Comment by Jolanrensen** [comment]
I agree, it's important for the library to be written as expected. Renamings like these will probably take a lot of time, however. 
We're looking into a good linter/formatter that can also detect stuff like this, however, with the size of the library we get flooded with hundreds of issues when we enable such a thing. It's probably best to go one file at a time and gradually reformat and rename stuff.

**Comment by Jolanrensen** [comment]
It's been a while since I've come across an incorrectly camel cased name. Feel free to reopen this issue if we missed one :)
